src/sass/_profile_detail_panel.scss

```diff
@@ -0,0 +1,89 @@
+.profile-detail-panel {
```

Comment:
I guess most of this CSS is identical to the instance detail panel. With a shared structure component, this would probably vanish.